test/src/tests-eCommerce.js

```diff
@@ -767,11 +818,14 @@ describe('eCommerce', function() {
         checkoutEvent.data.product_action.products[1].should.have.property('id', 'galaxySKU');
 
         done();
+        })
     });
 
     it('should log checkout option', function(done) {
+        waitForCondition(hasIdentifyReturned)
+        .then(() =>  {
         const product = mParticle.eCommerce.createProduct('iPhone', '12345', 400);
-        mockServer.requests = [];
+
```

Comment:
looks like we do a `findEventFromRequest` instead of looking at fetchMock.calls()[some index] to find it.  so this was not necessary.

i'll add a cleanup item that is something like "for anywhere we are looking at fetchMock.calls()[index], try using a helper method to get the event instead".  it is more robust in case the order of things ever change.